### DISCUSSION TITLE
Why the sticky-top in chrome get a line space when maximize chrome and turn on the bookmark menu?

### DISCUSSION THREAD

**Discussion by WROIATE** [twbs/bootstrap#31405] [General]
![Snipaste_2020-08-03_13-40-44](https://user-images.githubusercontent.com/44677306/89149726-b744cb00-d58f-11ea-80d5-949e7b0b4745.png)

According to this image, there hava a line space under the bookmark menu when I maximize chrome
When I close bookmark menu it will disappear

![Snipaste_2020-08-03_13-50-53](https://user-images.githubusercontent.com/44677306/89150095-5cf83a00-d590-11ea-82f6-f42347c56f58.png)

The same site in Firefox works well

![Snipaste_2020-08-03_13-59-26](https://user-images.githubusercontent.com/44677306/89150630-91202a80-d591-11ea-91a6-160de07800bf.png)

**Comment by taufik-nurhidayat** [comment]
what is your chrome version? in my chrome it work well but if i use responsive develepment tools it look same like your issues.

**Reply by WROIATE** [reply]
Version 84.0.4147.105 (Official Build) (64-bit)

**Reply by taufik-nurhidayat** [reply]
it's the same as the version I used. Ahh it looks like the problem occurred with bootstrapping.

**Comment by WROIATE** [comment]
Does anybody have some solutions?

**Reply by Deckluhm** [reply]
Can you provide a repro?

When you need help (or report a bug), always provide a repro so it's easier for others to help you.

**Reply by WROIATE** [reply]
@Deckluhm 
Sure. Here is an example
```
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
    <title>test</title>
</head>

<body>

    <header style="position:relative">
        <div class="jumbotron jumbotron-fluid bg-brightness mb-0" style="height: 30vh;">
        </div>
    </header>

    <div class="sticky-top p-0 mb-5 mr-0 navfadeIn">
        <div style="position: relative;">
            <div class="d-flex align-items-center justify-content-center" style="
                width: 100%;
                background-color: #050606;
                color: rgb(217, 215, 213);
                min-height: 70px;">
                <div style="width: 500px;">
                    <nav class="nav nav-pills flex-sm-row justify-content-center m-auto">
                        <a class="flex-sm-fill text-sm-center nav-link active" href="#">1</a>
                        <a class="flex-sm-fill text-sm-center nav-link " href="#">2</a>
                        <a class="flex-sm-fill text-sm-center nav-link " href="#">3</a>
                    </nav>
                </div>
            </div>
        </div>
    </div>

    <main style="height: 3000px; background-color: brown;">

    </main>

</body>
<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
    integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
    crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
    integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
    integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
    crossorigin="anonymous"></script>

</html>
```

![Snipaste_2020-08-08_22-38-57](https://user-images.githubusercontent.com/44677306/89713108-05aafd00-d9c8-11ea-8ef3-9cc826ecad17.png)
![Snipaste_2020-08-08_22-39-10](https://user-images.githubusercontent.com/44677306/89713109-0774c080-d9c8-11ea-8bba-680408401d1a.png)

**Reply by Deckluhm** [reply]
I can't reproduce your issue in both Firefox and Chromium on latest versions (Ubuntu 20.04).

Few things to try:

- Disable all browser addons
- Use latest Bootstrap version (you are using 4.3.1 and latest version if 4.5.2)

I have the feeling this isn't an issue with Bootstrap tho, if the issue just occurred with Chrome bookmark bar (but disappears without it), it might be a browser issue (or browser addon).
